environment.yml

```diff
@@ -27,6 +28,7 @@ dependencies:
 - pytest
 - pytest-flake8
 - nose
+- nbval
```

Comment:
Don't think we need `nbval` test library in production environment.  You already add `nbval` in `requirements_dev.txt`.